content/pages/datamanagement/demo_scandb.rst

```diff
@@ -21,10 +22,13 @@ series. This can be achieved using the following configuration::
            index-autofield-documenttype = datasets
            default-mode = autofield
 
-With this setup the DataLad `search` command will automatically discover
+With this setup the DataLad `search` command will automatically discovers
```

Comment:
Confused by this change. "discover" is right, isn't it?

Reply:
Fixed.